### PR TITLE
Total Autocorrect Overhaul

### PR DESCRIPTION
**Pull request by armorheart**
- Added a metric ton of new words
- Organized & Separated the autocorrect list into sets
- Refined existing words
- Removed insensible or irellevant words

----------------------------------------------------------------

POST PQ REMOVAL UPDATE : 
- Returned some missing words per requests
- Adjusted words to be grammatically correct

### CODE REVIEW

File: strings/accent_universal.json [hunk 1]
```diff
@@ -1,47 +1,94 @@
...
+		"wednesday": "wodnesdaeg",
+		"thursday": "thursdaeg",
+	    "lantern": "lamptern",
+		"stfu": "silence",
+		"skibidi": "fool",
+		"skibi": "fool",
+		"skib": "fool",
+		"ohio": "underworld",
+		"fanum": "Demon",
+		"cenat": "Demon",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:20]: uncapitalize please

File: strings/accent_universal.json [hunk 2]
```diff
@@ -1,47 +1,94 @@
...
-		"humour": "humor",
-		"saviour": "savior",
-		"the earth": "creation",
-		"earth": "creation",
-		"penis": "cock",
-		"retarded": "a nimrod",
-		"retard": "nimrod",
+		"allah": "gods",
+		"retarded": "foolish nimrod",
+		"retard": "fool nimrod",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:27]: r3tard3d = foolish
r3tard = nimrod

File: strings/accent_universal.json [hunk 3]
```diff
@@ -1,47 +1,94 @@
...
-		"copetti": "OINK OINK",
-		"jew": "merchant",
-		"day": "dae",
-		"night": "nite",
-		"years": "yils",
-		"wolf": "volf",
+		"jew": "merchant",	
+		"boomer": "dotard",
+		"boomers": "dotard",
+		"fazbear": "fae",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:42]: should be dire bear instead, it's better

File: strings/accent_universal.json [hunk 4]
```diff
@@ -1,47 +1,94 @@
...
+		"gamer": "gambler",
+		"bruh": "sire",
+		"fax": "truth",
+		"goated": "hallowed",
+		"dope": "gaunt",
+		"FOMO": "fear of gods",
+		"stan": "worshipper",
+		"stanning": "worshipping",
+		"cunny": "mechery",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:51]: lechery, not mechery

File: strings/accent_universal.json [hunk 5]
```diff
@@ -1,47 +1,94 @@
...
+		"rizz": "charm",
+		"rizzed": "charmed",
+		"rizzard": "charmer",
+		"rizzler": "charmer",
+		"gyatt": "buttocks",
+		"sussy": "suspicious",
+		"sus": "suspicious",
+		"baka": "nimrod",
+		"ligma": "leprosy",
+		"griddy": "charm",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:63]: fits better if changed to "gallop"

File: strings/accent_universal.json [hunk 6]
```diff
@@ -1,47 +1,94 @@
...
+		"banban": "demon",
+		"pog": "gleem",
+		"pogger": "gleeming",
+		"poggers": "gleeming",
+		"fap": "self abuse",
+		"fapping": "self abuse",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:69]: self abusING

File: strings/accent_universal.json [hunk 7]
```diff
@@ -1,47 +1,94 @@
...
+		"yap": "prattle",
```
Comment (ChungusGamer666) [inline strings/accent_universal.json:70]: yapping = prattling

**Review by ChungusGamer666 — CHANGES_REQUESTED** [review]
Almost perfect, please address the fails

File: strings/accent_universal.json [hunk 8]
```diff
@@ -1,47 +1,94 @@
...
+		"pussy": "cunt-bitten coward",
 		"vibing": "selfsucking",
-		"freak": "freek",
 		"weirdo": "weerdoe",
-		"human": "humen",
-		"sodomy": "zodomy",
-		"sodomite": "zodomite",
-		"sodomize": "zodomize",
-		"insane": "inzane",
-		"satan": "zizo",
```
Comment (TopHatPenguin) [inline strings/accent_universal.json:44]: Should probably keep satan as zizo

File: strings/accent_universal.json [hunk 9]
```diff
@@ -1,47 +1,94 @@
...
-		"retarded": "a nimrod",
-		"retard": "nimrod",
+		"allah": "gods",
+		"retarded": "foolish nimrod",
+		"retard": "fool nimrod",
 		"negro": "gnerd",
 		"nigger": "gnerd",
 		"nigga": "gnerd",
 		"niggo": "gnerd",
-		"lantern": "lamptern",
```
Comment (JTGSZ) [inline strings/accent_universal.json:23]: prob don't want to remove lantern - lamptern since its an actual item bro

Comment (armorheart) [reply strings/accent_universal.json:23]: Added it back,. Thanks!

File: strings/accent_universal.json (hunk 1 above)
Comment (armorheart) [reply strings/accent_universal.json:20]: Fixed

File: strings/accent_universal.json (hunk 8 above)
Comment (JTGSZ) [reply strings/accent_universal.json:44]: also he correct too zizo is a v prevalent thing

File: strings/accent_universal.json (hunk 2 above)
Comment (armorheart) [reply strings/accent_universal.json:27]: Fixed

File: strings/accent_universal.json (hunk 3 above)
Comment (armorheart) [reply strings/accent_universal.json:42]: Fixed

File: strings/accent_universal.json (hunk 4 above)
Comment (armorheart) [reply strings/accent_universal.json:51]: Fixed

File: strings/accent_universal.json (hunk 5 above)
Comment (armorheart) [reply strings/accent_universal.json:63]: Fixed

File: strings/accent_universal.json (hunk 6 above)
Comment (armorheart) [reply strings/accent_universal.json:69]: Fixed

File: strings/accent_universal.json (hunk 7 above)
Comment (armorheart) [reply strings/accent_universal.json:70]: Fixed

File: strings/accent_universal.json (hunk 8 above)
Comment (armorheart) [reply strings/accent_universal.json:44]: re-added as described

Comment (JTGSZ) [reply strings/accent_universal.json:44]: also since we here replacin satan with zizo, could also auto-translate lucifer to zizo

**Comment by TopHatPenguin** [comment]
At some point it should be decided upon if the server should carry on pick and choosing from both old English and middle English, or simply choose to to go fully into one.

File: strings/accent_universal.json (hunk 8 above)
Comment (armorheart) [reply strings/accent_universal.json:44]: Ok. Gonna hit it with another push

**Comment by dessysalta** [comment]
This changes "from" to "truthfullyom".

**Pull request merged**